penny_u_django/home/templates/home/index.html

```diff
@@ -3,10 +3,19 @@
 
 {% block content %}
 
-    <nav class="navbar navbar-light">
+    <nav class="navbar navbar-dark">
```

Comment:
IMO we should keep the navbar light.

Reply:
I agree that navbars should be light. I have several accessibility reasons for that.